src/registry_schemas/schemas/restoration.json

```diff
@@ -59,6 +59,10 @@
                         "registrar"
                     ]
                 },
+                "legalName": {
```

Comment:
Why wouldn't this be like the other legalName defs?
eg.
```
            "properties": {
                "legalName": {
                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/business#/definitions/legalName"
```

Reply:
Hey @thorwolpert Yes, totally agree, we shouldn't allow submissions to include a `legalName` field.  Are these schemas for submissions only -- not responses?  If so, is there somewhere you'd like me to update the API documentation?  I don't want to be *that* developer that adds a new attribute and doesn't let tell his colleagues.  :-)

Reply:
Responses include the business object in the header section of the filing.
This filing does not dictate the legalName, so it's not part of the return either.

Reply:
as a result of the restoration filing, the businesses name is changed.

Reply:
Understood, thanks for the explanation.  I'll close this PR.